### PR TITLE
Refactor even more wooden supports

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp [hunk 1]
```diff
@@ -647,7 +653,8 @@ static void AirPoweredVerticalRCTrackVerticalSlopeUp(
             PaintAddImageAsChildRotated(
                 session, direction, trackImageId, { 0, 0, height }, { { 0, 6, height }, { 20, 32, bbHeight } });
 
-            WoodenASupportsPaintSetup(session, 0, 0, height, session.SupportColours);
```
Comment (tupaschoal) [inline src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp:650]: Is this right? Unlike the other calls, the second argument of this one was not using `direction & 1`

Comment (Gymnasiast) [reply src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp:650]: Yes, this is correct - I replaced the `& 1` versions with `...Rotated(` and this occurrence with the regular version.

**Review by tupaschoal — APPROVED** [review]
I didn't test, but it looks ok as far as I could follow, because it's a bit big and easy to miss stuff

**Comment by ZeeMaji** [comment]
I haven't tested absolutely every ride type modified by this, but i looked at about 12 different parks and their wooden supports seem to be rendering correctly on various ride types.

**Review by ZehMatt — APPROVED** [review]

**Comment by ZeeMaji** [comment]
Scratch that, as soon as i made that comment i just noticed a minor regression on the virginia reel small turn pieces.

With this PR
Screenshot_select-area_20240216154935

Before
Screenshot_select-area_20240216154941

**Comment by ZehMatt** [comment]
> Scratch that, as soon as i made that comment i just noticed a minor regression on the virginia reel small turn pieces.
> 
> With this PR Screenshot_select-area_20240216154935
> 
> Before Screenshot_select-area_20240216154941

Good catch, I actually tested it as well and didn't spot anything.

File: src/openrct2/ride/coaster/VirginiaReel.cpp [hunk 2]
```diff
@@ -451,20 +476,14 @@ static void PaintVirginiaReelStation(
...
-    WoodenASupportsPaintSetup(session, (direction & 1), 0, height, session.SupportColours);
+    WoodenASupportsPaintSetupRotated(
+        session, WoodenSupportType::Truss, WoodenSupportSubType::NeSw, direction, height, session.SupportColours);
     TrackPaintUtilDrawStation(session, ride, direction, height, trackElement);
 
     PaintUtilSetSegmentSupportHeight(session, SEGMENTS_ALL, 0xFFFF, 0);
     PaintUtilSetGeneralSupportHeight(session, height + 32, 0x20);
 }
 
-static constexpr uint8_t virginia_reel_left_quarter_turn_supports[] = {
```
Comment (Gymnasiast) [inline src/openrct2/ride/coaster/VirginiaReel.cpp:461]: I suspect the problem lies here.

**Comment by Gymnasiast** [comment]
@ZeeMaji Thanks, I fixed it now.

**Pull request merged**